pkg/api/batch/warnings.go

```diff
@@ -0,0 +1,38 @@
+package batch
+
+import (
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/kubernetes/pkg/apis/batch"
+	"k8s.io/kubernetes/pkg/features"
+)
+
+var TTLWarningMsg = "This job doesn't have TTLAfterFinished field and not managed by a higher level controller. " +
```

Comment:
should be a const, lowercase (local to the batch package).

"and not" -> "and is not"